src/mono/wasm/runtime/debug.ts

```diff
@@ -294,6 +294,55 @@ export function mono_wasm_release_object(objectId: string): void {
         delete _call_function_res_cache[objectId];
 }
 
+export function mono_wasm_debugger_log(level: number, message_ptr: CharPtr): void {
+    const message = Module.UTF8ToString(message_ptr);
+
+    if (INTERNAL["logging"] && INTERNAL.logging["debugger"]) {
```

Comment:
maybe check that debugger/trace are functions to avoid crashing in the log functions if someone misconfigures it